lighthouse-core/report/html/report-styles.css

```diff
@@ -83,6 +83,9 @@
   --circle-size: calc(3 * var(--header-font-size));
 
   /* TODO(cjamcl) clean up unused variables. */
+  --audits-margin-bottom: 40px;
+  --circle-size-big: 120px;
+  --circle-size: 96px;
```

Comment:
~~what are these doing here/should we remove the `circle-size` above it?~~

oh looks like they're just alphabetized now :) still remove the one duplicate above it?

Reply:
done